dev-client/src/components/projects/ProjectListView/index.tsx

```diff
@@ -46,38 +46,43 @@ export default function Index({projects}: Props) {
 
       {projects.length > 0 && (
         <>
-          <HStack alignContent="center">
-            <VStack>
+          <HStack alignContent="center" mb="25px">
+            <Box paddingRight="13px" paddingTop="3px">
```

Comment:
question: Why use `px` here? Is it just for more accuracy for a smaller component?